### ISSUE TITLE
make geom_who names consistent

### ISSUE THREAD

**Issue by schloerke** [HBGDki/hbgd#2]
geom_who should be more like gg_who.  geom means you can "plus" into it... not supply a plot object

**Comment by hafen** [comment]
Yes thanks I was actually going to ask you to do this.  We want it to behave like a geom I just didn't have time and this was before the extensibility stuff was added.  We want it to be like a geom because it will usually be used in conjunction with other layers.
